build-logic/src/main/kotlin/Common.kt

```diff
@@ -37,6 +37,8 @@ private fun Project.configureTestAggregation() {
       attribute(USAGE_ATTRIBUTE, objects.named(Usage::class.java, "apolloTestAggregation"))
     }
   }
+  // Hide this from the 'assemble' task
+  configuration.setVisible(false)
```

Comment:
This is a fix for a build regression introduced in https://github.com/apollographql/apollo-kotlin/pull/5181 that made `:apollo-gradle-plugin:testJava11` depend on `:apollo-*:test`